tests/q.sql

```diff
@@ -0,0 +1,5 @@
+set max_threads=1;
```

Comment:
`tests/q.sql`? Shouldn't it be inside `tests/queries/0_stateless`?

Reply:
Oops, those test are not from this PR... How did I not notice it... Thanks